build/tasks/build.js

```diff
@@ -291,9 +291,27 @@ module.exports = function( grunt ) {
 	// Becomes:
 	//
 	//   grunt build:*:*:+ajax:-dimensions:-effects:-offset
+	//
+	// There's also a special "slim" alias that resolves to the jQuery Slim build
+	// configuration:
+	//
+	//   grunt custom:slim
 	grunt.registerTask( "custom", function() {
 		const args = this.args;
-		const modules = args.length ? args[ 0 ].replace( /,/g, ":" ) : "";
+		const modules = args.length ?
+			args[ 0 ]
+				.split( "," )
+
+				// Replace "slim" with respective exclusions meant for
+				// the official slim build
+				.reduce( ( acc, elem ) => acc.concat(
```

Comment:
`flatMap` would be so useful here... Alas, it's Node 12+ and we'll have to support at least v10 for the time being.